services/web/server/src/simcore_service_webserver/api/v0/openapi.yaml

```diff
@@ -4540,7 +4540,7 @@ paths:
         '403':
           description: ProjectInvalidRightsError
         '404':
-          description: UserDefaultWalletNotFoundError, ProjectNotFoundError
```

Comment:
i think there is an issue with the generator. The order in this description is random and every time we run it, it results in a different order. Perhaps we can add some sorting?